Linked Accounts/Link-Accounts.ps1

```diff
@@ -67,6 +64,7 @@ $URL_Logoff = $URL_Authentication + "/Logoff"
 # -----------
 $URL_Accounts = $URL_PVWAAPI + "/Accounts"
 $URL_LinkAccounts = $URL_PVWAAPI + "/Accounts/{0}/LinkAccount"
+$URL_Version = $URL_PIMServer + "/Server"
```

Comment:
@bab29 you have the same functionality also in /api/sever
Don't think you need to add the WebServices URL for that

Reply:
Using the PIMServers.svc is what is in documentation. At what point was it added to /api/server?

Reply:
10.3 if I'm not mistaken